Source/WebCore/plugins/PluginData.cpp

```diff
@@ -32,6 +32,8 @@
 
 namespace WebCore {
 
+static inline bool isBuiltInPDFPlugIn(const PluginInfo&);
```

Comment:
Where is this implemented?

Reply:
Later in the file. I'll move to this location, since it's a small function.